From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Thu, 21 May 2026 19:29:20 -0500
Subject: [PATCH] dusklight-flatpak

---
 CMakeLists.txt | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cf69a8e2ac..c24c968365 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -235,6 +235,11 @@ endif ()
 
 include(FetchContent)
 
+find_package(cxxopts 3.3)
+find_package(fmt 11.0)
+find_package(nlohmann_json 3.12)
+find_package(zstd)
+
 # Declare all dependencies first so CMake can download them in parallel
 message(STATUS "dusklight: Fetching cxxopts")
 FetchContent_Declare(cxxopts
@@ -329,7 +334,7 @@ set(GAME_LIBS aurora::core aurora::gx aurora::gd aurora::si aurora::vi aurora::p
     aurora::card freeverb cxxopts::cxxopts absl::flat_hash_map nlohmann_json::nlohmann_json TracyClient fmt::fmt
     Threads::Threads)
 
-list(APPEND GAME_LIBS libzstd_static)
+list(APPEND GAME_LIBS zstd)
 
 if (DUSK_ENABLE_SENTRY_NATIVE)
     list(APPEND GAME_LIBS sentry)
